java-frontend/src/main/java/org/sonar/java/se/checks/XxeProperty.java

```diff
@@ -81,24 +89,28 @@ static boolean isSetToTrue(@Nullable SymbolicValue sv1, ExpressionTree arg1) {
     private final BiPredicate<SymbolicValue, ExpressionTree> unsecuring;
     private final Constraint unsecured;
 
+    private final Constraint named;
+
     public XxePropertyHolder(String propertyName,
       BiPredicate<SymbolicValue, ExpressionTree> securing, Constraint secured,
-      BiPredicate<SymbolicValue, ExpressionTree> unsecuring, Constraint unsecured) {
+      BiPredicate<SymbolicValue, ExpressionTree> unsecuring, Constraint unsecured, Constraint named) {
```

Comment:
I would put `named` on the first line, next to `propertyName` to group them semantically.